server.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const exphbs = require('express-handlebars');
+const app = express();
+const bodyParser = require('body-parser');
+const PORT = 3000;
+
+const articles = require(`./routes/articles.js`);
+const products = require(`./routes/products.js`);
+
+app.engine('.hbs', exphbs({
+  defaultLayout: 'main',
+  extname: '.hbs',
+  layoutsDir: __dirname + '/views/layouts/',
+  partialsDir: __dirname + '/views/partials/'
+}));
+app.set('view engine', '.hbs');
+
+
+app.use(express.static('public'))
+
+app.use(
+  bodyParser.urlencoded({
+    extended: true,
+  }),
+);
+
+app.use('/products', products);
+app.use('/products/:id', products);
```

Comment:
You don't need these routes defined here:
```
app.use('/products/:id', products);
app.use('/products/:id/edit', products);
app.use('/products/new', products);
```
Anything that matches `/products` as a prefix from line 27 will be passed to your `products` router module.